scripts/node-cert-exporter.wrapper

```diff
@@ -6,10 +6,4 @@
 # Start the exporter with config options
 exec "${SNAP}/bin/node-cert-exporter" \
     $(safe_add_option listen) \
-    $(safe_add_option path) \
-    $(safe_add_option exclude-path) \
-    $(safe_add_option include-glob) \
-    $(safe_add_option exclude-glob) \
-    $(safe_add_option tls) \
-    $(safe_add_option tls-cert-file) \
-    $(safe_add_option tls-key-file)
+    --include-glob "/etc/ovn/cert_host,/etc/neutron/plugins/ml2/cert_host"
```

Comment:
I agree that we should match to `--path` to whatever we defined in the `plugs`. However, I will still keep `include-glob` / `exclude-glob` / `exclude-path` configurable, since the option `--path` has limitation that it will only consider the certificate with certain extensions [0,1]. Those options can allow user to include something like `/etc/ovn/cert_host,/etc/neutron/plugins/ml2/cert_host`

[0] https://github.com/amimof/node-cert-exporter/blob/master/pkg/exporter/exporter.go#L19C1-L19C1
[1] https://github.com/amimof/node-cert-exporter/blob/master/pkg/exporter/exporter.go#L115